hack/e2e-common.sh

```diff
@@ -297,3 +310,16 @@ function capture_cluster_logs() {
         exit 1
     fi
 }
+
+function get_vips() {
+  # Return vip at given index
+  idx=${1:-1} 
+  vips="${SHARED_DIR}/vips.txt"
```

Comment:
Similar here for `$SHARED_DIR`, though the `-f` below ought to catch if it's unset (unless we get unlucky and there's a `/vips.txt` in the file system...)